{
  try {
    return def.getTotalIn();
  }
 catch (  Exception e) {
    return 0;
  }
}
